qiita_ware/commands.py

```diff
@@ -70,22 +71,41 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
     ebi_submission.write_xml_file(ebi_submission.generate_submission_xml(),
                                   ebi_submission.submission_xml_fp)
 
-    # other steps
     if send:
-        ebi_submission.send_sequences()
-        study_accession, submission_accession = ebi_submission.send_xml()
+        # step 4: sending sequences
+
+        # Set the ASCP password to the one in the Qiita config, but remember
+        # the old pass so that we can politely reset it
+        old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
```

Comment:
That's correct. Basically I'm trying to make it as modular as possible (small PRs) as things change as I start figuring out what the previous version did and if they are wrong fixing them. 

BTW, from the original PR submission:
``After this the next step will be to (1) add the new-functional encrypted configuration file and (2) add the calls to moi and test in commands.py.```